core/templates/dev/head/components/summary_tile/exploration_summary_tile_directive.html

```diff
@@ -96,4 +102,9 @@ <h3 class="activity-title protractor-test-exp-summary-tile-title">
   .oppia-activity-summary-tooltip .tooltip-inner {
     min-width: 200px;
   }
+
+  .grayout {
+    opacity: 0.5;
+    filter: alpha(opacity = 50); /*IE*/
```

Comment:
I don't understand this -- the comment needs to be less cryptic. `opacity` doesn't work in IE? If so, we might have to change it in more places than just here...